docs/sms-send-how-to.md

```diff
@@ -42,6 +44,10 @@ grpcurl \
   messaging.MessageCore.SendTextMessage
 ```
 
+### Send binary messages
+
+Unfortunately, `grpcurl` only allows to send JSON formatted strings, so it can't be used to send binary SMS.
```

Comment:
Maybe add a "please have a look at the code example below" or something ?

Reply:
Done